extensions/wikia/DesignSystem/scripts/DesignSystemLoadingSpinner.js

```diff
@@ -0,0 +1,34 @@
+define('ext.wikia.design-system.loading-spinner', [
+	'jquery',
+	'ext.wikia.design-system.templating'
+], function ($, templating) {
+	'use strict';
+
+	var templateLocation = 'extensions/wikia/DesignSystem/services/templates/DesignSystemLoadingSpinner.mustache';
+
+	function LoadingSpinner(radius, strokeWidth) {
+		var self = this;
+		this.radius = radius || 30;
+		this.strokeWidth = strokeWidth || 6;
+		this.spinnerClasses = 'wds-spinner wds-block';
+		this.strokeClasses = 'wds-path';
+
+		this.init = function () {
+			return templating.renderByLocation(
```

Comment:
Are we sure we want to lazy load loading spinner?
I think something went wrong here.